### ISSUE TITLE
Better clipping on FB

### ISSUE THREAD

**Issue by weitzman** [joshkoenig/hillaryclintonisa_____butimvotingforheranyway.com#7]
When Facebook clips this site, its going to look for the first image and not find one. I think thats bad. The homepage really needs an image and a short description that looks good on FB. It shouldn't be overly reluctant either :)

Don't think I will submit a PR for this, so I reluctantly leave you with just a suggestion.

**Comment by joshkoenig** [comment]
Agreed. [My original inspiration](https://web.archive.org/web/20040723070151/http://www.johnkerryisadouchebagbutimvotingforhimanyway.com/) had this image/text Kerry header:

![image](https://cloud.githubusercontent.com/assets/143186/18443270/f7dc49c2-78c8-11e6-8d90-3126f3d8ea61.png)

I will try to find/create something similar.

**Comment by weitzman** [comment]
Yeah, that great. Bust out your meme maker app.

**Comment by gravelpot** [comment]
FWIW, I don't _think_ the page needs an actual image on it as long as it has the og:image meta tag with a URL to an image: http://ogp.me/

**Comment by joshkoenig** [comment]
Thanks. I'm going to try and get some changes done this weekend and see if I can get some traction coming out of the debates. Appreciate the help!
